### PR TITLE
You can now dig through sofa pillows

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

Worthless coins galore, you can find iron coins (Or something special) under sofa pillows by rightclicking with an empty hand.

## Why It's Good For The Game

Fun interaction with no side effects whatsoever

## Changelog

:cl:
add: You can now rightclick with an empty hand to search sofa pillows for change
/:cl:

### CODE REVIEW

File: modular_zubbers/code/game/objects/structures/beds_chairs/sofa.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	balloon_alert(user, "Searching under pillows...")
+	to_chat(user, span_alert("You start scouring through the sofa's pillows...."))
+	if(do_after(user, 10 SECONDS, src))
+		if(prob(40))
+			balloon_alert(user, "Found something")
+			if(prob(0.02)) // 0.02 About 1 in 125000, astronomically low
+				// Run through all nuclear disks and steal the first one
+				for(var/obj/item/disk/nuclear/N in SSpoints_of_interest.real_nuclear_disks)
+					do_teleport(N, src, channel = TELEPORT_CHANNEL_QUANTUM, forced = TRUE)
+					return
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/game/objects/structures/beds_chairs/sofa.dm:14]: I'm going to discuss this. The main PR itself is fine as a minor fluff thing, but I don't really like adding random astronomically low meme chances to "find" a critical station item. While extremely funny, this is on the table and I'm going to grab other's opinions before merging it.

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]
I asked and the agreement was no.

**Comment by YakumoChen** [comment]
Make it a fake disk

**Review by StrangeWeirdKitten — APPROVED** [review]

**Review by StrangeWeirdKitten — APPROVED** [review]

**Comment by StrangeWeirdKitten** [comment]
Oh I already approved it, lol. I'll look at merging this later

**Pull request merged**